Failure to build Unreal plugin on Linux due to inconsistent indentation

**System Information**
- Steam Audio version: 4.8.1
- (If applicable) Unreal Engine version: 5.6.1
- Operating System and version: Arch Linux
- (Optional) CPU architecture (e.g. x86-64, armv7): x86-64

**Issue Description**
Failure to build on Linux
```
[15/28] Compile Module.SteamAudio.cpp
In file included from /jenkins/workspace/Ech8 Dev Builds Linux/UnrealProject/Plugins/SteamAudio/Intermediate/Build/Linux/x64/UnrealEditor/Development/SteamAudio/Module.SteamAudio.cpp:42:
/jenkins/workspace/Ech8 Dev Builds Linux/UnrealProject/Plugins/SteamAudio/Source/SteamAudio/Private/SteamAudioSourceComponent.cpp:219:2: error: misleading indentation; statement is not part of the previous 'if' [-Werror,-Wmisleading-indentation]
  219 |         if ((InProperty->GetFName() == GET_MEMBER_NAME_CHECKED(USteamAudioSourceComponent, ReflectionsType)))
      |         ^
/jenkins/workspace/Ech8 Dev Builds Linux/UnrealProject/Plugins/SteamAudio/Source/SteamAudio/Private/SteamAudioSourceComponent.cpp:217:5: note: previous statement is here
  217 |     if ((InProperty->GetFName() == GET_MEMBER_NAME_CHECKED(USteamAudioSourceComponent, TransmissionHighValue)))
      |     ^
1 error generated.
```

**Steps To Reproduce**
Steps to reproduce the behavior:
1. Try to build on Linux

--- a/unreal/src/SteamAudioUnreal/Plugins/SteamAudio/Source/SteamAudio/Private/SteamAudioSourceComponent.cpp
+++ b/unreal/src/SteamAudioUnreal/Plugins/SteamAudio/Source/SteamAudio/Private/SteamAudioSourceComponent.cpp
@@ -216,16 +216,16 @@ bool USteamAudioSourceComponent::CanEditChange(const FProperty* InProperty) cons
         return bParentVal && !bSimulateTransmission;
     if ((InProperty->GetFName() == GET_MEMBER_NAME_CHECKED(USteamAudioSourceComponent, TransmissionHighValue)))
         return bParentVal && !bSimulateTransmission;
-	if ((InProperty->GetFName() == GET_MEMBER_NAME_CHECKED(USteamAudioSourceComponent, ReflectionsType)))
-		return bParentVal && bSimulateReflections;
-	if ((InProperty->GetFName() == GET_MEMBER_NAME_CHECKED(USteamAudioSourceComponent, CurrentBakedSource)))
-		return bParentVal && bSimulateReflections && (ReflectionsType == EReflectionSimulationType::BAKED_STATIC_SOURCE);
-	if ((InProperty->GetFName() == GET_MEMBER_NAME_CHECKED(USteamAudioSourceComponent, PathingProbeBatch)))
-		return bParentVal && bSimulatePathing;
-	if ((InProperty->GetFName() == GET_MEMBER_NAME_CHECKED(USteamAudioSourceComponent, bPathValidation)))
-		return bParentVal && bSimulatePathing;
-	if ((InProperty->GetFName() == GET_MEMBER_NAME_CHECKED(USteamAudioSourceComponent, bFindAlternatePaths)))
-		return bParentVal && bSimulatePathing;
+    if ((InProperty->GetFName() == GET_MEMBER_NAME_CHECKED(USteamAudioSourceComponent, ReflectionsType)))
+        return bParentVal && bSimulateReflections;
+    if ((InProperty->GetFName() == GET_MEMBER_NAME_CHECKED(USteamAudioSourceComponent, CurrentBakedSource)))
+        return bParentVal && bSimulateReflections && (ReflectionsType == EReflectionSimulationType::BAKED_STATIC_SOURCE);
+    if ((InProperty->GetFName() == GET_MEMBER_NAME_CHECKED(USteamAudioSourceComponent, PathingProbeBatch)))
+        return bParentVal && bSimulatePathing;
+    if ((InProperty->GetFName() == GET_MEMBER_NAME_CHECKED(USteamAudioSourceComponent, bPathValidation)))
+        return bParentVal && bSimulatePathing;
+    if ((InProperty->GetFName() == GET_MEMBER_NAME_CHECKED(USteamAudioSourceComponent, bFindAlternatePaths)))
+        return bParentVal && bSimulatePathing;
 
     return bParentVal;
 }

